src/main/java/core/basesyntax/service/RegistrationException.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service;
+
+public class RegistrationException extends RuntimeException {
+
+    public RegistrationException() {
+        super();
+    }
+
+    public RegistrationException(String message) {
+        super(message);
+    }
+
+    public RegistrationException(String message, Throwable cause) {
+        super(message, cause);
+    }
+
+    public RegistrationException(Throwable cause) {
+        super(cause);
+    }
```

Comment:
KISS principle - no need to implement things that you dont use 
```suggestion
    public RegistrationException(String message) {
        super(message);
    }
```